modules/virt-importing-vm-wizard.adoc

```diff
@@ -72,13 +72,22 @@ ifdef::virt-importing-rhv-vm[]
 . Optional: You can select *Start virtual machine on creation*.
 endif::[]
 
-. Optional: Click *Edit* to update the following settings:
+. Click *Edit* to update the following settings:
 
 ifdef::virt-importing-rhv-vm[]
 * *General* -> *Name*: The VM name is limited to 63 characters. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857165[(*BZ#1857165*)]
 * *General* -> *Description*: Optional description of the VM.
 * *Storage* -> *Storage Class*: The default storage class must be NFS.
```

Comment:
The default storage doesn't have to be NFS.
In CNV-2.4 only NFS is supported,  so for VM import from RHV, either need to pick NFS as target, or set NFS as default storage.

Reply:
I will reword this and tag it for 2.4 only. 

What is supported for 2.5?